### PR TITLE
required expressions added

### PR DESCRIPTION
**Pull request by Pranay10rawat**
Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by avsingh999** [comment]
@Pranay10rawat  great work ,
thanks :)

File: app/views/wiki/_wikis.html.erb [hunk 1]
```diff
@@ -9,6 +10,7 @@
   <% @wikis.each do |wiki| %>
     <tr>
       <td style="width:50%"><i class="fa fa-file"></i> <a href="<%= wiki.path %>"><%= wiki.latest.title %></a></td>
+      <td><i><%=wiki.path%></i></td>
```
Comment (avsingh999) [inline app/views/wiki/_wikis.html.erb]: @Pranay10rawat  please give space, like this
 `<td><i><%= wiki.path %></i></td>`

**Comment by jywarren** [comment]
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab
cc @jywarren @gauravano

**Pull request closed without merging**